03-task-manager/starter/package.json

```diff
@@ -3,19 +3,20 @@
   "version": "1.0.0",
   "description": "",
   "main": "app.js",
-  "scripts": {
-    "start": "nodemon app.js"
-  },
-  "keywords": [],
+ "scripts": {
+  "start": "node app.js",
+  "dev": "nodemon app.js"
+},
+ "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "mongoose": "^8.8.4"
+    "mongoose": "^5.11.10"
```

Comment:
I have to wonder, why downgrade Mongoose? Having older versions generally opens your project up to more well-known vulnerabilities.